solutions/gold/cf-510C.mdx

```diff
@@ -0,0 +1,110 @@
+---
+id: cf-510C
+source: CF
+title: Fox And Names
+author: Daniel Ge
+---
+
+## Solution
```

Comment:
Ok, I think you should rephrase the writing here to maybe lexicographically sort the strings or rephrase the writing so that it makes more sense.

Reply:
I'll rephrase the wording as you would like